src/EditorFeatures/CSharpTest/ChangeSignature/RemoveParametersTests.cs

```diff
@@ -293,11 +293,11 @@ class C
 
                 var handler = new CSharpChangeSignatureCommandHandler(TestWaitIndicator.Default);
                 var delegatedToNext = false;
-                Func<CommandState> nextHandler = () =>
+                CommandState nextHandler()
                 {
                     delegatedToNext = true;
                     return CommandState.Unavailable;
-                };
+                }
```

Comment:
These don't really change the allocation story, since a delegate will still get instantiated when passed to a GetCommandState, right?